server/src/main/java/io/spine/server/entity/storage/EntityRecordWithColumns.java

```diff
@@ -110,20 +114,38 @@ public EntityRecord getRecord() {
         return record;
     }
 
-    public Map<String, EntityColumn> getColumns() {
-        return Maps.transformEntries(storageFields,
-                                     ColumnTransformer.INSTANCE);
+    /**
+     * Obtains sorted {@linkplain EntityColumn#getStoredName() names}
+     * of entity columns for the record.
+     *
+     * <p>The names will be sorted using {@link String#compareTo(String)}.
+     *
+     * @return the sorted entity column names
+     */
+    public Collection<String> getColumnNames() {
```

Comment:
`Collection` is not usually returned in these cases.

Also, sorting each time this method is invoked is not too efficient. Maybe we could find an implementation of `storageFields`, which would have its keys sorted automatically.

Reply:
Done.